assets/js/hocs/with-searched-products.js

```diff
@@ -42,7 +42,11 @@ const withSearchedProducts = createHigherOrderComponent(
 			componentDidMount() {
 				const { selected } = this.props;
 				getProducts( { selected } )
-					.then( ( list ) => {
+					.then( ( products ) => {
+						const list = products.map( ( product ) => ( {
+							...product,
+							parent: 0,
+						} ) );
```

Comment:
hmm, this seems a _bit_ fragile.  I'm assuming the assumption here is that `getProducts` only returns products (not product variations) hence the adding of `parent:0` to get a consistent product shape?

I wonder, if instead this object shape massaging should happen in the `getProducts` call itself?  I think it'd be worthwhile doing that in this pull.

Reply:
Done in bfe3d4805de9e3a6320b1def6495fb194ba35a8d.